shell/platform/tizen/flutter_tizen.cc

```diff
@@ -28,7 +28,9 @@ static FlutterDesktopEngineRef HandleForEngine(
 FlutterDesktopEngineRef FlutterDesktopRunEngine(
     const FlutterDesktopWindowProperties& window_properties,
     const FlutterDesktopEngineProperties& engine_properties) {
-  flutter::StartLogging();
+#ifndef __X64_SHELL__
```

Comment:
Can't we run this at x64? :eyes:

Reply:
On the desktop shell, it's not necessary to run logging threads because the standard output and standard error are already available in the console (they only need to be redirected when using dlog). Actually the app process hangs when `Logger::Start` is called on x64 (I don't exactly know why).